City-of-Helsinki/berth-reservations-admin
# VEN-632 | Fix refetchQueries errors after assign/delete berths

## Description :sparkles:
- Use the query's name as a string in `refetchQueries`, in order to use the same variables used in the last operation.

## Issues :bug:
See the [conversation in Slack](https://helsinkicity.slack.com/archives/CBA1QHTHN/p1589969426031000)

## Testing :alembic:

### Manual testing :construction_worker_man:
- Go to the Applications page
- Assign a new berth by following the offer process
- The applications list data should be refetched reflecting the newly assigned berth
- Try to delete an assigned application
- The applications list data should be refetched reflecting the changes
